.github/workflows/release.yml

```diff
@@ -18,12 +18,19 @@ jobs:
   linux:
     runs-on: ${{ matrix.platform.runner }}
     strategy:
+      fail-fast: false
       matrix:
         platform:
           - runner: ubuntu-latest
             target: x86_64
           - runner: ubuntu-latest
             target: x86
+          - runner: ubuntu-latest
```

Comment:
Can you please update the cargo and project.toml file to 1.0.2 😅

Reply:
Can't Maturin read the version number from Cargo.toml?

Reply:
I didn't see a way to do that so unless I'm blind I feel not...

Reply:
It's listed in both dynamic and directly, that's incorrect. Will try removing from pyproject.toml, it says in the docs that they are merged.